modules/odc-customizing-user-perspective-using-CLI-and-UI.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+[id="odc-customizing-user-perspective_{context}"]
+= Customizing user perspective 
+
+The OCP web console provides two default perspectives, namely *Administrator* and *Developer*. You might have more perspectives available depending on installed console plugins. As a cluster administrator, you can show or hide a perspective to all users or a specific user role. Customizing the perspective ensures that users can view only the perspective that is applicable to their role and tasks, which avoids confusion. For example, hide the *Administrator* perspective from unprivileged users so that they cannot manage cluster resources, users, and projects or show the *Developer* perspective to users with the developer role so that they can create, deploy, and monitor applications. 
+
+You can also customize the perspective to show to or hide from users based on Role-based access control (RBAC). For example, if you customize a perspective for monitoring purposes, which requires specific permissions, you can define that the perspective is visible only to users with required permissions.
+
+Each perspective includes the following mandatory parameters, which you can edit in the YAML view:
+
+* `id`: Defines the ID of the perspective to show or hide
+* `visibility`: Defines the state of the perspective along with access review checks, if needed 
+* `state`: Defines whether the perspective is enabled, disabled, or needs an access review check
+
+[NOTE]
+====
+By default, all perspectives are enabled. When you customize the user perspective, your changes are applicable to the entire cluster.
+====
+
+.Prerequisites
+* You must have administrator privileges.
+
+.Procedure to customize perspective using YAML view
+
+. In the *Administrator* perspective, navigate to *Administration* -> *Cluster Settings*.
+. Select the *Configuration* tab and click the *Console (operator.openshift.io)* resource.
+. Click the *YAML* tab and make your customization: 
+.. To enable or disable a perspective, insert the snippet for *Add user perspectives* and edit the YAML code as needed:
++ 
+[source,yaml]
+----
+spec:
```

Comment:
The contrib guide say  to always include:

> Required fields
> Include the apiVersion and kind so that a user always knows the context of the YAML.
> 
> Include the full hierarchy to a deeply nested key.
> 
> For objects that are in the global scope, such as for config.openshift.io API group, always include the metadata.name for the object, which is usually cluster.

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#yaml-formatting-for-kubernetes-and-openshift-api-objects